bin/sdc-dirty-vms

```diff
@@ -180,7 +181,7 @@ function getPackages(callback)
     var packages = {};
     var package_uuid;
 
-    child = spawn(cmd, args, {'customFds': [-1, -1, -1]});
+    child = spawn(cmd, args, {sdtio: [-1, -1, -1]});
```

Comment:
Is that a typo?  Or is it really "sdtio"?

Reply:
You're right, that is a typo.